core/src/test/scala/org/apache/spark/sql/delta/DeltaCDCSuite.scala

```diff
@@ -964,3 +964,11 @@ class DeltaCDCScalaSuite extends DeltaCDCSuiteBase {
   }
 
 }
+
+class DeltaCDCScalaWithDeletionVectorsSuite extends DeltaCDCScalaSuite
```

Comment:
does this suite cover the streaming tests?

Reply:
Nope... but I just added one!